### PR TITLE
updated docs titles

### PR DESCRIPTION
**Pull request by AndrewChubatiuk**
### Describe Your Changes

https://victoriametrics-rocks.slack.com/archives/C01T9QWAE0P/p1722513382481579

### Checklist

The following checks are **mandatory**:

- [ ] My change adheres [VictoriaMetrics contributing guidelines](https://docs.victoriametrics.com/contributing/).

### CODE REVIEW

**Review by dmitryk-dk — CHANGES_REQUESTED** [review]
Please update the title for cloud

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

File: docs/Articles.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ See also [case studies](https://docs.victoriametrics.com/casestudies/).
 * [Miro: Prometheus High Availability and Fault Tolerance strategy, long term storage with VictoriaMetrics](https://medium.com/miro-engineering/prometheus-high-availability-and-fault-tolerance-strategy-long-term-storage-with-victoriametrics-82f6f3f0409e)
 * [ZERODHA: Infrastructure monitoring with Prometheus at Zerodha](https://zerodha.tech/blog/infra-monitoring-at-zerodha/)
 * [ZERODHA: Monitoring K8S with VictoriaMetrics](https://docs.google.com/presentation/d/1g7yUyVEaAp4tPuRy-MZbPXKqJ1z78_5VKuV841aQfsg/edit)
-* [Criteo: VictoriaMetrics, a stress-free Prometheus Remote Storage for 1 Billion metrics](https://medium.com/criteo-engineering/victoriametrics-a-prometheus-remote-storage-solution-57081a3d8e61)
+* [Criteo: VictoriaMetrics, a stress-free Prometheus Remote Storage for 1 Billion metrics](https://github.com/zhihu/promate)
```
Comment (hagen1778) [inline docs/Articles.md:31]: why this link is replaced?

File: docs/Release-Guide.md [hunk 2]
```diff
@@ -91,8 +91,7 @@ Bumping the limits may significantly improve build speed.
    and this release contains all the needed binaries and checksums.
 1. Update the release description with the content of [CHANGELOG](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/docs/CHANGELOG.md) for this release.
 1. Publish release by pressing "Publish release" green button in GitHub's UI.
-1. Bump version of the VictoriaMetrics cluster in the [sandbox environment](https://github.com/VictoriaMetrics/ops/blob/main/gcp-test/sandbox/manifests/benchmark-vm/vmcluster.yaml)
```
Comment (hagen1778) [inline docs/Release-Guide.md:94]: I'd keep this link even if it is unreachable for public users.
wdyt @zekker6 @f41gh7 ? Do you think it somehow exposes us from sec perspective?

File: docs/anomaly-detection/components/reader.md [hunk 3]
```diff
@@ -53,13 +53,13 @@ reader:
 
 ### Per-query parameters
 
-Starting from [v1.13.0](/anomaly-detection/changelog/v1130) there is change of [`queries`](/anomaly-detection/components/reader?highlight=queries#vm-reader) arg format. Now each query alias supports the next (sub)fields:
+Starting from [v1.13.0](/anomaly-detection/changelog#v1130) there is change of [`queries`](/anomaly-detection/components/reader?highlight=queries#vm-reader) arg format. Now each query alias supports the next (sub)fields:
```
Comment (hagen1778) [inline docs/anomaly-detection/components/reader.md:56]: @fred-navruzov @Cambaza it is recommended using absolute links in documentation - see this guideline https://docs.victoriametrics.com/#documentation

File: docs/data-ingestion/README.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 In This Folder you will find instructions for sending data to VictoriaMetrics from a variety of platforms.
-If your tool is not listed it is likely you can ingest your data into VictoriaMetrics using one of the protocols listed in our [Prominent features]({{< ref "/Single-server-VictoriaMetrics.md#prominent-features" >}}) section.
+If your tool is not listed it is likely you can ingest your data into VictoriaMetrics using one of the protocols listed in our [Prominent features](../#prominent-features) section.
```
Comment (hagen1778) [inline docs/data-ingestion/README.md:2]: @tiny-pangolin  it is recommended using absolute links in documentation - see this guideline https://docs.victoriametrics.com/#documentation

File: docs/Articles.md (hunk 1 above)
Comment (AndrewChubatiuk) [reply docs/Articles.md:31]: in a description thread there's a link to a thread with google doc, where almost all these changes are defined

Comment (hagen1778) [reply docs/Articles.md:31]: I don't see https://medium.com/criteo-engineering/victoriametrics-a-prometheus-remote-storage-solution-57081a3d8e61 link in that doc. 

And besides the doc, this replacement doesn't seem correct at all.

File: docs/Release-Guide.md (hunk 2 above)
Comment (zekker6) [reply docs/Release-Guide.md:94]: This hypothetically gives more information about our environment, but I don't see any issues with that. There is no way to get unauthorized access to this link anyway, so there is no harm in keeping it in public repo.

If there are concerns regarding keeping this link as a public one I would suggest adding a release guide doc in ops repo and add reference to this doc here.

File: docs/Articles.md (hunk 1 above)
Comment (AndrewChubatiuk) [reply docs/Articles.md]: it was a typo, updated it

File: docs/Release-Guide.md (hunk 2 above)
Comment (AndrewChubatiuk) [reply docs/Release-Guide.md:94]: @hagen1778 @zekker6 what do you think about moving it from docs, that a published to a website to READMEs which are only in a repo?

Comment (hagen1778) [reply docs/Release-Guide.md:94]: Makes sense to me. Do we reference this links somewhere else? Can it be removed safely?

File: docs/data-ingestion/README.md (hunk 4 above)
Comment (hagen1778) [inline docs/data-ingestion/README.md:2]:
```suggestion
If your tool is not listed it is likely you can ingest your data into VictoriaMetrics using one of the protocols listed in our [Prominent features](https://docs.victoriametrics.com/#prominent-features) section.
```

**Review by hagen1778 — APPROVED** [review]

**Pull request merged**